packages/icrc-ledger-types/src/icrc1/account.rs

```diff
@@ -308,4 +363,54 @@ mod tests {
             Err(ICRC1TextReprError::InvalidChecksum { expected: _ })
         );
     }
+
+    #[test]
+    fn test_account_max_serialization_length() {
```

Comment:
Do we want to proptest this?

Reply:
In a followup PR (draft: https://github.com/dfinity/ic/pull/1513) it will be indirectly proptested by [test_upgrade_serialization](https://github.com/dfinity/ic/blob/be1fe485efa07a896a09da4d0854d65e7ffdc933/rs/ledger_suite/tests/sm-tests/src/lib.rs#L2575), however, I can also see the point of having a dedicated proptest...

Reply:
I added the proptest to the test below. The code for `account_strategy` is not duplicated with `test_utils`, I created    https://dfinity.atlassian.net/jira/software/c/projects/FI/boards/110?selectedIssue=FI-1536 to address that.